.github/workflows/wrapper-validation.yaml

```
@@ -1,6 +1,6 @@
 name: Gradle Wrapper Validation ✅
 
-on: [pull_request]
+on: [ pull_request ]
```

```suggestion
on: [pull_request]
```

A space after and before the bracket does not represent the style guidelines for Github actions.